spec/requests/api/v1/allocations_controller_index_spec.rb

```diff
@@ -167,6 +167,50 @@
           expect(response.body).to eq('{"error":{"locations":["may not be used in combination with `from_locations` or `to_locations` filters."]}}')
         end
       end
+
+      describe 'included relationships', :skip_before do
+        let!(:allocations) { create_list :allocation, 2, :with_moves }
+
+        before do
+          get "/api/v1/allocations#{query_params}", params: params, headers: headers
+        end
+
+        context 'when not including the include query param' do
+          let(:query_params) { '' }
+
+          let(:included) do
+            from_locations = allocations.map(&:from_location)
+            to_locations = allocations.map(&:to_location)
+            moves = allocations.flat_map(&:moves)
+
+            from_locations + to_locations + moves
+          end
+
+          it 'returns the default includes' do
+            expect(response_json).to have_includes(included)
+          end
+        end
+
+        context 'when including the include query param' do
+          let(:query_params) { '?include=foo.bar,from_location' }
```

Comment:
We've explicitly agreed to defer handling of invalid includes errors for the moment and AMS doesn't propagate any errors/leave any visible sign of some failure.

```markdown
# Inclusion of Related Resources
An endpoint MAY return resources related to the primary data by default.

An endpoint MAY also support an include request parameter to allow the client to customize which related resources should be returned.

If an endpoint does not support the include parameter, it MUST respond with 400 Bad Request to any requests that include it.

If an endpoint supports the include parameter and a client supplies it, the server MUST NOT include unrequested resource objects in the included section of the compound document.

The value of the include parameter MUST be a comma-separated (U+002C COMMA, “,”) list of relationship paths. A relationship path is a dot-separated (U+002E FULL-STOP, “.”) list of relationship names.

If a server is unable to identify a relationship path or does not support inclusion of resources from a path, it MUST respond with 400 Bad Request.
```